### ISSUE TITLE
[a11y]: Progress Bar - Example -  Loading Message

### ISSUE THREAD

**Issue by guidari** [carbon-design-system/carbon#16594]
### Package

@carbon/react

### Package version

v1.58.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

Exporting Data Loading status not conveyed by Screen Reader
Once we hit on Example Link and press enter OR space bar Exporting Data Loading Message starts but Screen Reader doesn't announce the loading status for exporting / Complete. Visually it shows 20MB, 50MB etc till 720MB.

### Recommended Fix
Screen Reader should instantly announce the loading status for the user.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-progressbar--example

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
The issue was retested by our accessibility team (in @carbon/react v1.84), and the issue has been fixed.

> Now screen reader is announcing loading state:
> JAWS speech history:
> 135.7MB of 728MB
> Done
